apps/docs/src/guide/transactions/transaction-request.md

```diff
@@ -30,31 +30,57 @@ A `CreateTransactionRequest` is used for create transactions, which are transact
 
 Once you have instantiated a transaction request, you can modify it by setting the transaction parameters and policies. This can either be done manually by directly altering the transaction request object, or through helper methods that are available on the above classes.
 
-### Adding Resources to a Transaction Request
+### Adding `OutputCoin`
 
-Resources populate the inputs and outputs of a transaction request. This can take the form of coins, messages or contracts. The SDK provides a range of methods for dealing with resources. Below will detail how coins and messages can be added to a transaction request.
+Including `OutputCoins` in the transaction request specifies the UTXOs that will be created once the transaction is processed. These UTXOs represent the amounts being transferred to specified account addresses during the transaction:
 
-<<< @./snippets/transaction-request/fund-request.ts#transaction-request-3{ts:line-numbers}
+<<< @./snippets/transaction-request/add-output-coin.ts#transaction-request-3{ts:line-numbers}
 
-### Adding a Contract to a Transaction Request
+### Estimating and Funding the Transaction Request
 
-Scripts can perform multiple actions on chain, therefore you may want to chain contract calls. For this you will need to add a contract to the transaction request. This can be done like so:
+Before submitting a transaction, it is essential to ensure it is properly funded to meet its requirements and cover the associated fee:
 
-<<< @./snippets/transaction-request/fund-request.ts#transaction-request-4{ts:line-numbers}
+<<< @./snippets/transaction-request/estimate-and-fund.ts#transaction-request-4{ts:line-numbers}
+
+This is the recommended approach for manually estimating and funding a transaction before submission. It ensures that the `gasLimit` and `maxFee` are accurately calculated, and that the required amounts for `OutputCoins` are fulfilled. The `fund` method automatically fetches any missing resource amounts from the calling account and adds them to the transaction request.
+
+### Manually Fetching Resources
+
+In certain scenarios, you may need to manually fetch resources. This can be achieved using the `getResourcesToSpend` method, which accepts an array of `CoinQuantities` and returns the necessary resources to meet the specified amounts:
+
+<<< @./snippets/transaction-request/fetch-resources.ts#transaction-request-5{ts:line-numbers}
+
+> **Note**: While manually fetching and adding resources to a transaction request is possible, it is generally not recommended as it potentially leads to funding errors.
+
+#### Manually Fetching Coins or Messages
+
+If needed, you can manually include specific coins or messages in the transaction. However, this approach is generally discouraged and should only be used in scenarios where explicitly adding particular coins or messages to the transaction request is required:
+
+<<< @./snippets/transaction-request/fetch-coins.ts#transaction-request-6{ts:line-numbers}
```

Comment:
The note and paragraph here seem to be saying the same thing, do we need the note?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3440/commits/c67f9ae09150c576c425809a96c03c07ff20a242